torchgeo/datasets/geo.py

```diff
@@ -611,12 +611,12 @@ def __getitem__(self, query: BoundingBox) -> Dict[str, Any]:
         )
         if shapes:
             masks = rasterio.features.rasterize(
-                shapes, out_shape=(int(height), int(width)), transform=transform
+                shapes, out_shape=(int(round(height)), int(round(width))), transform=transform
```

Comment:
This line is too long now. Can you run `black torchgeo/datasets/geo.py` to fix this?

Reply:
There is need to cast it to `int` again if you're not specifying the number of decimals to round to.

Reply:
Unless this was needed to satisfy mypy.